src/components/Integrations/Shopify/Shopify.test.js

```diff
@@ -152,4 +157,52 @@ describe('Shopify Component', () => {
     await waitForDomChange();
     expect(getByText('validation_messages.error_unexpected_HTML'));
   });
+
+  // Failing test indicating memory leak in shopify component
+  xit('should use DopplerAPI client when feature is enabled', async () => {
```

Comment:
xit?

Reply:
To skip this test  -->https://jestjs.io/docs/en/api#testskipname-fn